### ISSUE TITLE
no-empty rule does not work for class methods

### ISSUE THREAD

**Issue by unlight** [eslint/typescript-eslint-parser#96]
**What version of TypeScript are you using?**
1.8.10

**What version of `typescript-eslint-parser` are you using?**
0.3.0

**What code were you trying to parse?**
```ts
export class EditComponent {

    emptyFunc() { // line 9
    }

    ngOnInit() { 
        { // line 13
            // empty block
        }
    }
}
```

**What did you expect to happen?**
Rule `no-empty` Empty block statement should warn about `emptyFunc` on line 9.

**What happened?**
ESLint results:
```
13:9  warning  Empty block statement  no-empty
```

**Comment by unlight** [comment]
Sorry, missed `no-empty-function`
